sdk/communication/communication-administration/CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
+
+### Added
+
+- Added support for long-running operations. See details under Breaking Changes.
+
+### Breaking Changes
+
+- Renamed `PhoneNumberSearch` to `PhoneNumberReservation` and renamed its `searchId` property to `reservationId`.
+- Renamed `cancelSearch` to `cancelReservation`.
```

Comment:
I like the format that we used in Python that also explains which type owns the renamed methods. Should we do it here as well?

Reply:
Like this: https://github.com/Azure/azure-sdk-for-java/pull/17571/files

Reply:
I like your idea but I tried it and it feels more verbose and harder to read
![image](https://user-images.githubusercontent.com/1885005/99110903-f8da0e80-259f-11eb-8918-f92fa4984ab3.png)

We're also already more hierarchical with sections in our Changelog than other JS libraries that use [Breaking] prefixes instead of sections. Adding another section level for more groups feels like it reduces readability as it's overall more text as we split option renames away from client methods.

Reply:
Pushed a commit, let's do the grouping.